pegasus/sites.v3/code.org/views/lms/lms_integration.haml

```diff
@@ -0,0 +1,12 @@
+.action-block.action-block--two-col
```

Comment:
You can add a `.white` class here so it stands out against the gray bg. 
```suggestion
.action-block.action-block--two-col.white
```

Reply:
resolved!